dapper/xp_launch.py

```diff
@@ -464,7 +464,7 @@ def launch(self, HMM, save_as="noname", mp=False,
             def xpi_dir(*args): return None
         else:
             save_as = rc.dirs.data / Path(save_as).stem
-            save_as /= "run_" + datetime.now().strftime("%Y-%m-%d__%H:%M:%S")
+            save_as /= "run_" + datetime.now().strftime("%Y-%m-%d__%H-%M-%S")
```

Comment:
I'm not really in favor of this, because I find the original format easier to read for humans. Any reason you suggested this change?

Reply:
I nearly forgot this change. I did this in an ad-hoc way as I'm running DAPPER in Windows (at least generating files in Windows) and the system does not allow colon in filenames.  It seems most Windows files with timestamp use %H%M%S without any symbols in between. I think we should either keep the original format or add one condition for Windows system, or simply delete the colons.

Reply:
Very good point! Windows should be supported, but I'm running on Mac and Linux, and so have a hard time testing with windows. I had this bug once: it seems in Windows you cannot have folders named "utils", which was funny.

I really want human-readability, and personally found it very hard to read without separators. I suggest we stick with your change, using dashes as separators.

BTW, others have had the same [question](https://old.reddit.com/r/ISO8601/comments/d6mdaz/iso_8601_compliant_filenames_on_windows/).

Reply:
Good, I will keep the change. I just tested and I think the quotation marks are the problem for "utils". It is always interesting to find out this kind of design differences.